protected/humhub/modules/admin/views/setting/oembed_edit.php

```diff
@@ -25,21 +86,30 @@
 
 <br>
 
-<?php $form = ActiveForm::begin(['id' => 'authentication-settings-form', 'acknowledge' => true]); ?>
+<?php $form = ActiveForm::begin(['id' => 'oembed-provider-form', 'acknowledge' => true]); ?>
 
 <?= $form->errorSummary($model); ?>
 
-<?= $form->field($model, 'prefix')->textInput(['class' => 'form-control']); ?>
-<p class="help-block"><?= Yii::t('AdminModule.settings', 'Url Prefix without http:// or https:// (e.g. youtube.com)'); ?></p>
+<?= $form->field($model, 'name')->textInput(['class' => 'form-control']); ?>
+
+<?= $form->field($model, 'pattern')->textInput(['class' => 'form-control']); ?>
+<p class="help-block"><?= Yii::t('AdminModule.settings', 'Regular expression by which the link match will be checked.'); ?></p>
 
 <?= $form->field($model, 'endpoint')->textInput(['class' => 'form-control']); ?>
 <p class="help-block"><?= Yii::t('AdminModule.settings', 'Use %url% as placeholder for URL. Format needs to be JSON. (e.g. http://www.youtube.com/oembed?url=%url%&format=json)'); ?></p>
 
+<?php if(isset($query['access_token'])): ?>
+    <?= $form->field($model, 'access_token')->textInput(['class' => 'form-control endpoint-param', 'data-param-name' => 'access_token', 'value' => $query['access_token']]) ?>
```

Comment:
Would it be possible to add an entry for `'type' => 'password'` to automatically hide the entered text here, or is this overkill @luke-?

Reply:
@ArchBlood Thanks, usually it makes sense, but the token is appended plaintext to the URL anyway, so I would ignore it in this case :-)